etc/greenboot/greenboot.conf

```diff
@@ -7,9 +7,12 @@
 ### This variable controls 
 ### This value can be "true, TRUE, True..." as it will be lowercased.
 ### Set it to anything else to disable this check.
-GREENBOOT_WATCHDOG_CHECK_ENABLED=true
+# GREENBOOT_WATCHDOG_CHECK_ENABLED=true
```

Comment:
is this disabled for testing purposes?

Reply:
this was kept like this, and now it's regressing in CI